bundles/redhat8.8/packages.txt.gotmpl

```diff
@@ -31,8 +31,8 @@ make
 libseccomp
 nfs-utils
 iproute-tc
-kernel-headers-4.18.0-477.58.1.el8_8
-kernel-devel-4.18.0-477.58.1.el8_8
+kernel-headers
```

Comment:
we no longer need to pin the kernel header versions?

If thats the case, we need similar fix for redhat-8.6

Reply:
in that case, should we create template variable for EUS support and toggle version accordingly?

Reply:
i think thats a bit too overkill for this. 

I think this is something that very few customers run into and the kernel version requirements should be documented. Whatever we set for our tests can be used just for that.